packages/venia-ui/lib/queries/mergeCarts.gql.js

```diff
@@ -0,0 +1,39 @@
+import gql from 'graphql-tag';
+
+import { CartPageFragment } from '../components/CartPage/cartPageFragments.gql';
+import { CartTriggerFragment } from '../components/Header/cartTriggerFragments.gql';
+import { CheckoutPageFragment } from '../components/CheckoutPage/checkoutPageFragments.gql';
+
+export const mergeCarts = gql`
```

Comment:
Mentioned elsewhere, but consider renaming this to follow our normal convention (assuming all caps constant style is our convention).

```suggestion
export const MERGE_CARTS = gql`
```

Reply:
I'll rename it to something more inline with the other gql exports like `mergeCartMutation`.